### PR TITLE
Fixed entry points stdout missing output

### PR DESCRIPTION
**Pull request by midnightercz**
Also fixed enum evaluation

### CODE REVIEW

File: src/pubtools/sign/signers/msgsigner.py [hunk 1]
```diff
@@ -435,13 +435,44 @@ def _msg_clear_sign(inputs, signing_key=None, task_id=None, config=None):
             str_inputs.append(input_)
     operation = ClearSignOperation(inputs=str_inputs, signing_key=signing_key, task_id=task_id)
     signing_result = msg_signer.sign(operation)
+    print(
```
Comment (querti) [inline src/pubtools/sign/signers/msgsigner.py:438]: test print?

File: src/pubtools/sign/signers/msgsigner.py [hunk 2]
```diff
@@ -435,13 +435,44 @@ def _msg_clear_sign(inputs, signing_key=None, task_id=None, config=None):
...
+    """Run containersign operation with cli arguments."""
+    msg_signer = MsgSigner()
+    config = _get_config_file(config)
+    msg_signer.load_config(load_config(os.path.expanduser(config)))
+
+    operation = ContainerSignOperation(
+        digests=digest, references=reference, signing_key=signing_key, task_id=task_id
+    )
+    signing_result = msg_signer.sign(operation)
+    print(
```
Comment (querti) [inline src/pubtools/sign/signers/msgsigner.py:462]: test print?

File: pyproject.toml [hunk 3]
```diff
@@ -63,6 +63,6 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project.scripts]
-pubtools-sign-msg-clear-sign = "pubtools.sign.signers.msgsigner:msg_clear_sign_main"
-pubtools-sign-msg-container-sign = "pubtools.sign.signers.msgsigner:msg_container_sign_main"
+pubtools-sign-msg-clear-sign = "pubtools.sign.signers.msgsigner:msg_clear_sign"
```
Comment (querti) [inline pyproject.toml:66]: The _main functions still exist, what is their purpose?

**Review by querti — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 3 above)
Comment (midnightercz) [reply pyproject.toml]: no, main functions are needed I just forgot to revert it here

**Review by querti — APPROVED** [review]

**Review by emilyzheng — APPROVED** [review]

**Pull request merged**